.travis.yml

```diff
@@ -141,12 +139,11 @@ jobs:
     script: skip
     deploy:
       provider: pypi
-      skip_cleanup: true
```

Comment:
This is important to keep. The warning says that it's `cleanup:` now

Reply:
Let’s revert this change given https://travis-ci.community/t/skip-cleanup-true-is-now-deprecated-how-to-go-for-gradle-publish/6641